src/commonMain/kotlin/io/github/oshai/kotlinlogging/KLoggingEventBuilder.kt

```diff
@@ -4,4 +4,13 @@ public class KLoggingEventBuilder {
   public var message: String? = null
   public var cause: Throwable? = null
   public var payload: Map<String, Any?>? = null
+  public var internalCompilerData: InternalCompilerData? = null
```

Comment:
please document that this field is internal and should not be set

Reply:
Added JavaDoc